tests/test_pytypes.cpp

```diff
@@ -209,6 +209,7 @@ TEST_SUBMODULE(pytypes, m) {
     m.def("get_tuple_from_iterable", [](const py::iterable &iter) { return py::tuple(iter); });
     // test_float
     m.def("get_float", [] { return py::float_(0.0f); });
+    m.def("cast_float", [](py::float_ f) { return f; });
```

Comment:
Could you please rename this to `roundtrip_float` or `float_roundtrip` or similar?

(`cast` is super ambiguous in the context pybind11; C/C++ cast, to-Python, from-Python, all in the same name bucket unfortunately)

Reply:
done